perf/README.md

```diff
@@ -26,13 +33,28 @@ Benchmark results can be visualized with https://observablehq.com/@libp2p-worksp
 5. `CLIENT_IP=$(terraform output -raw client_ip)`
 6. `SERVER_IP=$(terraform output -raw server_ip)`
 
-### Build and run implementations
+**Notes**
+- You may need to reset the infrastructure if you encounter any errors, you can do that by running `terraform destroy` and then `terraform apply`.
```

Comment:
I don't think this is a good general hint. In the most cases terraform should be able to reconciliate any changes without a previous `terraform destroy`. On a high level, I am hesitant to give general terraform advice here. I think that is much better placed in the official terraform documentation.